pygments/formatters/img.py

```diff
@@ -206,13 +206,17 @@ def get_char_size(self):
         """
         Get the character size.
         """
-        return self.fonts['NORMAL'].getsize('M')
+        return self.get_text_size('M')
 
     def get_text_size(self, text):
         """
-        Get the text size(width, height).
+        Get the text size (width, height).
         """
-        return self.fonts['NORMAL'].getsize(text)
+        font = self.fonts['NORMAL']
+        if hasattr(font, 'getbbox'):  # Pillow >= 9.2.0
+            return font.getbbox(text)[2:4]
```

Comment:
```suggestion
            return font.getlength(text)
```

Reply:
If I read the docs correctly, `getlength` only returns a single value, while `getsize` returns `(width, height)`.